### ISSUE TITLE
question about padding in conv2d

### ISSUE THREAD

**Issue by xslittlegrass** [thtrieu/darkflow#54]
In the file `net/ops/convolution.py`:

```
temp = tf.pad(self.inp.out, [[0, 0]] + pad + [[0, 0]])
temp = tf.nn.conv2d(temp, self.lay.w['kernel'], padding = 'VALID',  name = self.scope, strides = [1] + [self.lay.stride] * 2 + [1])
```

Is this equivalent to the 'same' padding,  `tf.nn.conv2d(..., padding ='SAME')`?

**Comment by jiangsutx** [comment]
I think they are the same.

I reimplement this yolo2 net by myself with the same pretrained weight. (slim.conv2d & padding='SAME')
The output is the same as Darkflow.

**Comment by xslittlegrass** [comment]
@jiangsutx Thanks!
